packages/mobile_app/lib/components/prophecy_record.dart

```diff
@@ -8,140 +8,308 @@ import 'package:language/language.dart';
+            prophecy.value.toStringAsFixed(1),
+            style: TextStyle(
+              fontSize: 22.0,
+              fontWeight: FontWeight.w400,
+              color: chooseNumberColor(prophecy.value),
             ),
           ),
         ],
       ),
-      decoration: BoxDecoration(
-          gradient: LinearGradient(
-            colors: [
-              AppColors.prophecyGradientStart.withOpacity(0.7),
-              AppColors.prophecyGradientEnd.withOpacity(0.7),
-            ],
-            begin: Alignment.topLeft,
-            end: Alignment.topRight,
-          ),
-          borderRadius: BorderRadius.circular(8.0)),
-    );
-  }
+    ),
+    decoration: BoxDecoration(
+        gradient: LinearGradient(colors: [
+          AppColors.prophecyValueProgressGradient[0].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[1].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[2].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[3].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[4].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[5].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[6].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[7].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[8].withOpacity(0.7),
+          AppColors.prophecyValueProgressGradient[8].withOpacity(0.7),
+          AppColors.prophecyGradientEnd.withOpacity(0.6),
+        ], stops: [
+          0.0,
+          0.1,
+          (valuePercent > 0.3) ? 0.3 : valuePercent,
+          (valuePercent > 0.4) ? 0.4 : valuePercent,
+          (valuePercent > 0.5) ? 0.5 : valuePercent,
+          (valuePercent > 0.6) ? 0.6 : valuePercent,
+          (valuePercent > 0.7) ? 0.7 : valuePercent,
+          (valuePercent > 0.8) ? 0.8 : valuePercent,
+          (valuePercent > 0.9) ? 0.9 : valuePercent,
+          (valuePercent > 1.0) ? 1.0 : valuePercent,
+          valuePercent,
+        ]),
+        borderRadius: BorderRadius.circular(8.0)),
+  );
 }
 
-class ProphecyTextWidget extends StatelessWidget {
-  final bool textExists;
-  final String text;
-  ProphecyTextWidget({@required this.textExists, this.text = ""});
+Container prophecyRecordExpanded(
+    {@required ProphecyEntity prophecy,
+    @required Map<bool, String> planetVariants}) {
+  final textExists = prophecy.text != null && prophecy.text.isNotEmpty;
 
-  @override
-  Widget build(BuildContext context) {
-    if (textExists) {
-      return Container(
-        padding: EdgeInsets.only(top: 8),
-        child: Wrap(
-          children: <Widget>[
-            Container(
-              margin: EdgeInsets.only(top: 8),
-              height: 116,
-              child: ListView(
+  double value = prophecy.value ?? 0.0;
+
+  var valuePercent = value / 10;
+
+  bool isNone = true;
+  bool isPositive = false;
+  String planetName = "";
+
+  if (value >= _HIGH) {
```

Comment:
Some simple explanations what is happening here wont hurt

Reply:
It is also temporary code I saved =)